.github/workflows/autoupdate.yml

```diff
@@ -9,11 +9,15 @@ jobs:
       - uses: actions/checkout@v3
         with:
           ssh-key: ${{ secrets.MINIFORGE_AUTOUPDATE_SSH_PRIVATE_KEY }}
-      - uses: conda-incubator/setup-miniconda@35d1405e78aa3f784fe3ce9a2eb378d5eeb62169
```

Comment:
Can we please fix this upstream ( https://github.com/conda-incubator/setup-miniconda/issues/248 )?